src/main/java/MaxSumMain.java

```diff
@@ -1,2 +1,44 @@
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.util.Scanner;
+
 public class MaxSumMain {
+
+    public static void main(String[] args) {
+        try {
+            //Read file
+            String filePath = args[0];
+            File myObj = new File(filePath);
```

Comment:
I think you may be able to do File(args[0]), instead of making a separate variable